uq.esys.escript.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+
+  <!-- Copyright 2019 Adam Ellery <a.ellery@uq.edu.au> -->
+  <id>uq.esys.escript</id>
```

Comment:
I think this app-id isn't ideal, should likely be more like au.edu.yq.esys.escript